exporters/trace/build.gradle

```diff
@@ -1,7 +1,12 @@
+plugins {
+    id 'java'
+}
+
 description = 'Cloud Trace Exporter for OpenTelemetry'
 
 dependencies {
     api(libraries.opentelemetry_api)
     api(libraries.opentelemetry_sdk)
     api(libraries.google_cloud_trace)
+		testImplementation('junit:junit:4.13')
```

Comment:
Indentation looks wrong

Reply:
mb. was using tabs and it looked the same to my editor